spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1149,10 +1201,10 @@ def main(args=None):
             printed = False
             while spot_ros.spot_wrapper.is_estopped():
                 if not printed:
-                    print('\033[93mWaiting for estop to be released.  Make sure you have an active estop.'
+                    print(COLOR_YELLOW + 'Waiting for estop to be released.  Make sure you have an active estop.'
```

Comment:
also ok to leave as print

Reply:
we have a node at this point, so we could use the logger.  Do you still want a raw `print`?

Reply:
I had a reason for that but I forget what it was...  Let's leave it as a print for now.